launcher/cloudbuild.yaml

```diff
@@ -1,7 +1,7 @@
 substitutions:
   '_BASE_IMAGE': '' # an empty base image means the build will use the latest image in '_BASE_IMAGE_FAMILY'
-  '_BASE_IMAGE_FAMILY': 'cos-113-lts' # base image family
-  '_OUTPUT_IMAGE_PREFIX': 'confidential-space'
+  '_BASE_IMAGE_FAMILY': 'cos-tdx-113-lts' # base image family
```

Comment:
nit: so I'm assuming `cos-tdx-113-lts` family will also work for SEV-based CS image?